src/clock-weather-card.ts

```diff
@@ -182,14 +182,16 @@ export class ClockWeatherCard extends LitElement {
 
   private renderForecast(): TemplateResult[] {
     const weather = this.getWeather();
+    // TODO swith to getTemperature, once merged to master
     const currentTemp = weather.attributes.temperature !== undefined ? Math.round(weather.attributes.temperature) : null;
-    const days = this.config.forecast_days;
+    const items = this.config.forecast_days;
+    const hourly = this.config.forecast_hourly;
```

Comment:
what happens if a user defines `forecast_hourly: true`, but the weather integration provides `daily` values? have you tried that?